src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const li = document.querySelectorAll('li');
```

Comment:
Consider using `document.querySelectorAll('li')` with a more specific selector if possible, to avoid selecting unintended `li` elements. This will improve performance and ensure the script targets only the intended elements.